src/renderer/Components/CreatingPackage/PackagePreview/Order/index.jsx

```diff
@@ -19,11 +26,22 @@ function Order({ order, index }) {
         </div>
       </div>
       <div className="mx-auto my-2 flex w-fit flex-row items-center justify-start justify-items-start space-x-4 rounded border border-solid border-gray-300 p-2">
-        {sourcePath && <div className="text-block-blue">{sourcePath} 의</div>}
-        <div className="text-block-blue">{attachmentName} 을(를)</div>
-        <div className="text-block-blue">{executionPath} 에</div>
-        {editingName && (
-          <div className="text-block-blue">{editingName} 으로</div>
+        {sourcePath && attachmentType !== "folder" && (
+          <div className="text-block-blue">{sourcePath} 의</div>
+        )}
```

Comment:
```suggestion
        {(sourcePath && attachmentType !== "folder") && (
          <div className="text-block-blue">{sourcePath} 의</div>
        )}
```

밑(38번째 줄)에서는 여기서도 소괄호를 씌워주고 있네요! 
어느 쪽이든 통일을 시켜주시는 게 좋을 것 같습니다 :)

Reply:
38 줄에서는 꼭 필요한 괄호이고 여기서는 있어도 되고 없어도 되는 괄호여서 안써주었는데여 주연님 말씀듣고 통일 시키려고 괄호를 사용하고 저장했더니 자동 프리티어 때문에 다시 벗겨지는 현상이 발생해서 기존 했던 대로 가야할 것 같습니다. 😁

Reply:
아이코 프리티어 설정이로군요... 알겠습니다 !